src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/renametool/SpecialRenameUserPage.java

```diff
@@ -0,0 +1,109 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.special.renametool;
+
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.SpecialPageObject;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.net.URLEncoder;
+
+public class SpecialRenameUserPage extends SpecialPageObject {
+  @FindBy(css = "input[name=\"newUsername\"]")
+  private WebElement newUsernameTextBox;
+  @FindBy(css = "input[name=\"newUsernameRepeat\"]")
+  private WebElement confirmNewUsernameTextBox;
+  @FindBy(css = "input[name=\"submitbutton\"]")
+  private WebElement submitButton;
+  @FindBy(xpath = "//a[text()=\"Staff log\"")
+  private WebElement staffLogLink;
+  @FindBy(css = ".errorbox")
+  private WebElement errorMessageTextBox;
+  @FindBy(css = "#mw-content-text")
+  private WebElement contentTextBox;
+  @FindBy(css = ".extiw")
+  private WebElement helpLink;
+  @FindBy(css = "#password")
+  private WebElement currentPasswordTextBox;
+  @FindBy(css = "#understand-consequences")
+  private WebElement termsAndConditionsCheckBox;
+  @FindBy(css = ".successbox")
+  private WebElement successBox;
+  @FindBy(css = "#mw-content-text")
+  private WebElement renamedMessage;
+
+
+  public SpecialRenameUserPage(WebDriver driver) {
+    super();
+    PageFactory.initElements(driver, this);
+  }
+
+  public SpecialRenameUserPage open() {
+    getUrl(urlBuilder.getUrlForWiki() + URLsContent.SPECIAL_RENAME_TOOL);
+    return this;
+  }
+
+  public SpecialStaffLogPage openStaffLog() {
+    wait.forElementVisible(staffLogLink);
+    staffLogLink.click();
+    return new SpecialStaffLogPage(driver);
+  }
+
+//  public SpecialRenameUserPage fillFormData(String s, String s1) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)